src/unclutter.c

```diff
@@ -147,6 +149,9 @@ static void parse_args(int argc, char *argv[]) {
                 } else if (OPT_NAME_IS("hide-on-touch")) {
                     config.hide_on_touch = true;
                     break;
+                } else if (OPT_NAME_IS("hide-on-kbd")) {
```

Comment:
Sure. I actually considered that but ended up going with "hide-on-kbd" because I was afraid that "key" without "keyboard" might be too ambiguous for users who aren't familiar with X11, and "hide-on-keyboard-press" feels a bit too long. I suppose "hide-on-kbd-press" isn't too much longer but I also doubt there will ever be a need to support key release. In any case, I'll leave these design choices to you. 👍

Reply:
Appreciate the thought you’re putting into it!